Framework/PythonInterface/plugins/algorithms/LoadVesuvio.py

```diff
@@ -142,13 +142,46 @@ def PyExec(self):
     def validateInputs(self):
         issues = {}
 
-        # Validtae run number ranges
+        # Validate run number ranges
         run_str = self.getProperty(RUN_PROP).value
         if "-" in run_str:
             lower, upper = run_str.split("-")
             if upper < lower:
                 issues[RUN_PROP] = "Range must be in format lower-upper"
 
+
+        # Validate SpectrumList
+        speclists_str = self.getProperty(SPECTRA_PROP).value
+
+        if ";" in speclists_str:
+            # Split if in 2-3;6-7 format
+            speclists_str = speclists_str.split(";")
+        else:
+            # Treat spec as a list
+            speclists_str = [speclists_str]
+
+        for speclist_str in speclists_str:
+            if "-" in speclist_str:
+                lower, upper = speclist_str.split("-")
+                lower = int(lower)
+                upper = int(upper)
+                # Check format
+                if upper < lower:
+                    issues[SPECTRA_PROP] = "Range must be in format lower-upper"
+                # Check Min/Max boundaries
+                if lower < 3:
```

Comment:
Good idea. This would probably mean that I would have to move this out of validation and into a separate function (similar to `_raise_error_if_non_valid_mode` and `_raise_error_if_mix_fwd_back`) as _load_common_inst_parameters() isn't called by that point. I'm reluctant to put `_load_common_inst_parameters()` the validate function as I believe it takes a couple of seconds to find/read the file.